dockerfiles/jupyter-for-local-hotreload/Dockerfile

```
@@ -1,32 +1,52 @@
+# Build with:
+#   docker build -t deepnote/jupyter-for-local:local -f dockerfiles/jupyter-for-local-hotreload/Dockerfile .
+
+ARG FROM_PYTHON_TAG=3.12
 FROM deepnote/python:${FROM_PYTHON_TAG}
 
+ARG FROM_PYTHON_TAG
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install system dependencies
 RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)